resources/views/ignored.blade.php

```diff
@@ -12,19 +15,15 @@
 
             <section class="bg-white rounded-b-lg">
                 @foreach ($projects as $project)
-                    <ul class="flex list-reset p-4 border-t border-smoke">
+                    <ul id="project_{{ $project->id }}" class="flex list-reset p-4 border-t border-smoke">
                         <li class="w-1/2">
                             <a class="text-indigo-700 hover:text-indigo-900 no-underline text-md" href="{{ $project->github_url }}">
                                 {{ $project->name }}
                             </a>
                         </li>
 
                         <li class="w-1/2 text-right">
-                            <form action="{{ route('project.unignore', $project) }}" method="POST">
-                                @method('PATCH')
-                                @csrf
-                                <input type="submit" value="Un-Ignore" class="bg-indigo-600 hover:bg-indigo-500 text-white px-3 py-1 rounded cursor-pointer">
-                            </form>
+														<button onClick="ignoreProject({{$project->id}}, false);" type="button" class="bg-indigo-600 hover:bg-indigo-500 text-white px-3 py-1 rounded cursor-pointer">Un-Ignore</button>
```

Comment:
Not so important but instead of passing an `ignore = false` flag here, can make a dedicated function called `unignoreProject(projectId)`? That would allow developers browsing the codebase  to not to have to think about what passing `false` here does.

I think keeping `ignoreProject()`'s signature the same and having `unignoreProject()` just call it with the boolean would good.

Reply:
Sure, make sense. Implemented.